application-accelerator/configuration.hbs.md

```diff
@@ -103,6 +123,64 @@ tanzu accelerator create my-hello-fun --git-repo https://github.com/sample-accel
 
 >**Note:** It is not currently possible to provide the `git.ignore` option with the Tanzu CLI.
 
+## <a id="creating-git-credentials"></a> Configuring a Git credentials secret to be used with non-public repositories and custom CA certificates
+
+When deploying accelerators using Git repositories that need authentication and/or are installed with custom CA certificates then you need to provide some additional autentication values in a Secret. The examples in the next section provide more details about this. In this section we describe how to conventiently configure a Git credentials secret that can be used for some of the Git based examples below.
+
+You can specify the following accelerator configuration properties when installing the Application Accelerator. The same properties can be provided in the `accelerator` section of the `tap-values.yaml` file:
+
+```yaml
+git_credentials:
+  secret_name: git-credentials
+  username: GIT-USER-NAME
+  password: GIT-PASSWORD-OR-ACCESS-TOKEN
```

Comment:
The expectation here is that an organization would have some sort of service account & its credentials would be entered here, is that right? If so, perhaps it would be beneficial to mention that "we recommend using a service account for this"?

Reply:
Added "We recommend using an access token for this."